test/spec/vanilla.spec.js

```diff
@@ -160,16 +172,17 @@ describe('backend with vanillajs', function() {
   it('should not serve up mock data when params do not match', function () {
```

Comment:
Maybe you're right, the matching/not matching tests kinda seem scattered (which is my fault), after we get all this merged I'll probably refactor the tests to make them a bit more organized.